apps/wpcom-block-editor/src/calypso/features/iframe-bridge-server.js

```diff
@@ -690,7 +693,7 @@ async function openLinksInParentFrame( calypsoPort ) {
 			}
 		}
 	} );
-	const popoverSlotElem = document.querySelector( '.interface-interface-skeleton ~ .popover-slot' );
+	const popoverSlotElem = document.querySelector( 'body' );
```

Comment:
This needs a better solution because running this code everytime the whole body element changes is probably a bad performance move.

Reply:
I'm not actually sure how much of a bad idea this is? Like is it bad, or terrible?

Reply:
It shouldn't be inefficient, according to the [MutationObserve docs](https://developer.mozilla.org/en-US/docs/Web/API/MutationObserver/observe) setting `childList: true` doesn't look for changes on all the child elements, it just checks for childs added or removed from the observed element. If we wanted to observe everything we would also need to supply `subtree: true`.